Source/WebCore/css/html.css

```diff
@@ -790,28 +790,34 @@ input:-webkit-autofill, input:-webkit-autofill-strong-password, input:-webkit-au
 input:is([type="radio"], [type="checkbox"]) {
     margin: 3px 2px;
     box-sizing: border-box;
-    border: initial;
+    border: -internal-auto-base(initial, 0.3em solid currentColor);
 #if defined(WTF_PLATFORM_IOS_FAMILY) && WTF_PLATFORM_IOS_FAMILY
-    width: 16px;
-    height: 16px;
-    padding: 0px;
+    width: -internal-auto-base(16px, 1em);
+    height: -internal-auto-base(16px, 1em);
+    padding: -internal-auto-base(0px, unset);
     /* We want to be as close to background:transparent as possible without actually being transparent */
     background-color: rgba(255, 255, 255, 0.01);
 #else
+    width: -internal-auto-base(unset, 1em);
+    height: -internal-auto-base(unset, 1em);
     padding: initial;
     background-color: initial;
 #endif
 }
 
 input[type="checkbox"] {
 #if defined(WTF_PLATFORM_IOS_FAMILY) && WTF_PLATFORM_IOS_FAMILY
-    border-radius: 5px;
+    border-radius: -internal-auto-base(5px, 0.5em);
+#else
+    border-radius: -internal-auto-base(unset, 0.5em);
```

Comment:
Doesn't match [the spec](https://drafts.csswg.org/css-forms/#basic-appearance-stylesheet). Does the PR need updating or the spec need updating or is there no decision here yet (in which case, do we have an issue filed)?